client/packages/portal-core/src/utils/utils.ts

```diff
@@ -0,0 +1,16 @@
+export function getPathUrl(path: string, contextId?: string): string {
+	if (!contextId) return `${path}/`;
+	return `${path}/${contextId}`;
+}
+
+export function getContextPageUrl(contextId?: string): string {
+	if (!contextId) return 'context-page/';
+	return `/context-page/${contextId}`;
+}
+
+export function getContextFormUrl() {
+	const match = window.location.pathname.match(
```

Comment:
Is regex really needed to get the context from url?

Reply:
This i s used outside on the routing so its ider this of sting splitting. and if checks